west.yml

```diff
@@ -0,0 +1,25 @@
+# The west manifest file for downstream Zephyr module ZXA.
+#
+# The per-installation west configuration file, .west/config, sets the
+# path to the project containing this file in the [manifest] section's
+# "path" variable.
+#
+# This imports the latest upstream Zephyr manifest file.
+
+manifest:
+  version: 0.7
+
+  defaults:
+    remote: zephyrproject-rtos
+
+  remotes:
+    - name: zephyrproject-rtos
+      url-base: https://github.com/zephyrproject-rtos
+
+  #
+  # Please add items below based on alphabetical order
+  projects:
+    - name: zephyr
+      remote: zephyrproject-rtos
+      revision: master
+      import: true
```

Comment:
I would consider adding a `path-prefix: zephyrproject-rtos` here, along with a comment. This would then place all zephyr-related repos in a folder, which goes along with your desire to keep the app separate from the rest of zephyr itself. So the repo would look like:

```
example-application/
├── app
├── zephyr
│   └── module.yml
└── zephyrproject-rtos
    ├── modules
    └── zephyr
```

Reply:
@carlescufi did you really mean to put `zephyrproject-rtos` underneath `example-application` here?

It should be in the same directory, like this, no?

```
workspace/
├── example-application
│   └── zephyr
│       └── module.yml
├── .west
└── zephyrproject-rtos
    ├── modules
    └── zephyr
```

Reply:
I assume you mean having the path reflect the location of the parent repo. That pattern means that users and scripts referencing `zephyr-env.sh` must look in multiple places for it (zephyrproject-rtos vs nordic vs other forks).  It also means an even longer path for users to type.  I see the costs, but I don't see the benefit. What am I missing?

Reply:
>It should be in the same directory, like this, no?
Indeed, fixed now

Reply:
>I assume you mean having the path reflect the location of the parent repo.

No, it's really about "encapsulating" the upstream project and modules in their own subfolder so it is visible what comes from upstream and what does not. It's only a suggestion though, and in part it's a matter of taste.